js/flow/src/telemetry.ts

```diff
@@ -144,19 +146,20 @@ export function writeFlowFailure(
     relevantPaths.forEach((p) => {
       pathCounter.add(1, {
         ...pathDimensions,
-        success: 'success',
+        status: 'success',
         path: p.path,
       });
 
       pathLatencies.record(p.latency, {
         ...pathDimensions,
+        status: 'success',
         path: p.path,
       });
     });
 
     pathCounter.add(1, {
```

Comment:
Should we be tracking failed path latency here?

Reply:
Added.